models/marts/enriched_history/enriched_history_operations.sql

```diff
@@ -1,12 +1,17 @@
-{{ config(
-    tags = ["enriched_history_operations"],
-    materialized='incremental',
-    unique_key=["op_id"],
-    partition_by={
+{% set meta_config = {
```

Comment:
What's the difference between `config` and `set meta_config`?

Reply:
I see `exports model config as meta tag so that we can query it from elementary schema. This is further used to calculate primary test coverage`